package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "./node_modules/karma/bin/karma start",
-    "build": "rm -rf lib && mkdir lib && node-sass src -o lib && babel src -d lib",
+    "build": "rm -rf lib temp && mkdir lib temp && node-sass src -o temp && css-bundler temp/style.css lib/style.css && babel src -d lib && rm -rf temp",
```

Comment:
`css-bundler` didn't *really* bundled the css files: the main file was still importing two other files from the `lib` directory. It may have been enough, but I prefer to have a all-in-one style file to be sure...